clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4956,7 +4886,68 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     }
   }
 
+#ifdef ENABLE_CLASSIC_FLANG
+  if (Args.getLastArg(options::OPT_fveclib))
+    Args.AddLastArg(CmdArgs, options::OPT_fveclib);
+  else
+    CmdArgs.push_back("-fveclib=PGMATH");
+#else
   Args.AddLastArg(CmdArgs, options::OPT_fveclib);
+#endif
+
+  std::string PassRemarkVal(""), PassRemarkOpt("");
```

Comment:
I think this whole block also needs an include guard as it uses `OPT_Minfo_EQ` and friends? So I think this branch currently wouldn't build with `-DLLVM_ENABLE_CLASSIC_FLANG=Off`

Reply:
Weird also how this block needs to be in the Clang driver and can't just be in ClassicFlang.cpp